### DISCUSSION TITLE
r/An-RV-user-advises-you-not-to-buy-an-RV

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#66] [Announcements]
# r/An-RV-user-advises-you-not-to-buy-an-RV

为什么我自己在用房车，还经常自驾房车旅行，并且写旅行游记出来分享，我却劝别人不要买房车？

https://blog.ops-coffee.cn/r/An-RV-user-advises-you-not-to-buy-an-RV

**Comment by AhuiZhao** [comment]
年检也得拆座椅。

**Reply by ops-coffee** [reply]
6年免检，过了6年看看
